recipes/uwebsockets/all/conanfile.py

```diff
@@ -27,21 +27,27 @@ def configure(self):
             "Visual Studio": "15",
             "gcc": "7",
             "clang": "5",
-            "apple-clang": "10"
+            "apple-clang": "10",
         }
 
         compiler = str(self.settings.compiler)
         if compiler not in minimal_version:
             self.output.warn(
-                "%s recipe lacks information about the %s compiler standard version support" % (self.name, compiler))
+                "%s recipe lacks information about the %s compiler standard version support"
```

Comment:
Did your formatter change these lines automatically?
Which one are you using?
Our .editorconfig allows 200 columns per line.

Reply:
@ruilvo Try to avoid formatting files when bumping versions, because it breaks `bump version` tag, and CCI bot is not smart enough to detect it.

Reply:
Ah damn, sorry, I didn't notice it did that! I just checked the file and I must have saved without paying attention. I use `black`, which is very opinionated as you might know. It just slipped. 
Want me to revert this?

Reply:
Nope, it's good enough. Black is an excellent but I doubt most of contributors are using it. We don't block PRs because formatting, it's more like a false-positive in terms of counting, like if we want to know how many `bump version` PRs we had last month.